Update run section in readme (#1076)
purescript/spago: README.md

$ spago build --before clear --before "notify-send 'Building'"
```

> If you want to run the program (akin to `pulp run`), just use `run`. The commands below are verbose until these values are defined in the configuration file:

Reviewer: I don't think there's a need to mention the verbosity since we'll explain that in a second
```suggestion
If you want to run the program, just use `run`:
```

```console
$ spago run -p package-name -m Module.Containing.Main

# We can pass arguments through to `purs compile`
$ spago run -p package-name  -m Module.Containing.Main --purs-args "--verbose-errors"

# We can pass arguments through to the program being run
$ spago run -p package-name  -m Module.Containing.Main -- arg1 arg2
```

> Oof! That's a lot to run a command. If you configure these parameters in the package's configuration file's `package.run` section, you don't have to supply them at the command line.
> See [here](#the-configuration-file) for more info about this, but it allows us to instead write.

Reviewer: Some wording to make it smoother to read
```suggestion
Oof! That's a lot of typing. Fortunately it's possible to configure most of these parameters in the `package.run` section of your configuration file, so you don't have to supply them at the command line.

See [here](#the-configuration-file) for more info about this, but it allows us to instead write:
```

> ```console
> # module defined
> spago run -p package-name
> spago run -p --purs-args "--verbose-errors"

Reviewer: The second command here doesn't work, so I think you mean to write this
```suggestion
```console
# The main module can be defined in the configuration file, but
# purs flags still need to be supplied at the command line
spago run -p package-name --purs-args "--verbose-errors"
```

> # Args passed to program defined
> $ spago run -p package-name

Reviewer: Think it's useful to show an example of the config in this case
```suggestion
# It's possible to even pass arguments from the config, which would look like this:
#
# package:
#   run:
#       main: Main
#       execArgs:
#         - "arg1"
#         - "arg2"
$ spago run -p package-name
```
